pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/assembly/AssemblyVersionUpload.java

```diff
@@ -81,7 +82,7 @@ protected void doRun(SlingHttpServletRequest request, PostResponse response, Lis
                     .draft().getOrCreate();
 
             // Check if the content is the same as what is hashed already
-            HashCode incomingSrcHash = hash(asciidocContent);
+            HashCode incomingSrcHash = JcrResources.hash(asciidocContent);
```

Comment:
Interesting that you chose to leave `asciidocContent` as a String here, rather than reworking it to be an InputStream instead. Was that intentional?

Reply:
The change to use a single checksum hash method is intentional, but I focused on the module uploading initially and not the assembly upload. Perhaps that is the next step after this is reviewed.